src/wp-includes/script-loader.php

```diff
@@ -273,7 +273,7 @@ function wp_default_packages_scripts( $scripts ) {
 	 *     'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
 	 *     'api-fetch.js' => array(...
 	 */
-	$assets = include ABSPATH . WPINC . '/assets/script-loader-packages.php';
+	$assets = include ABSPATH . WPINC . '/assets/script-loader-packages.min.php';
```

Comment:
It would be even better if we could have the following:

```suggestion
	$assets = include ABSPATH . WPINC . "/assets/script-loader-packages{$suffix}.php";
```

The issue I have is that when running unit tests on CI it resolves to the version without `.min` because the `SCRIPT_DEBUG` flag is enabled. The other file is only generated during development and ignored by version control so it causes fatal errors. The only way I can think about working around it would be:

```php
$assets = include ABSPATH . WPINC . '/assets/script-loader-packages.min.php';
if ( ! $suffix && file_exists( ABSPATH . WPINC . '/assets/script-loader-packages.php' ) ) {
    $assets = include ABSPATH . WPINC . '/assets/script-loader-packages.php';
}
```

What do you think?

Reply:
I see that `WP_RUN_CORE_TESTS` constant is used in a few places. I will try to use it here as well.